components/fieldgroup/package.json

```diff
@@ -19,18 +19,20 @@
     "@spectrum-css/checkbox": "^3.1.1",
     "@spectrum-css/icon": "^3.0.21",
     "@spectrum-css/radio": "^3.0.22",
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/helptext": "^1.0.20",
+    "@spectrum-css/tokens": "^1.0.1"
   },
   "devDependencies": {
+    "@spectrum-css/component-builder-simple": "^1.0.0-beta",
     "@spectrum-css/checkbox": "^3.1.3",
-    "@spectrum-css/component-builder": "^3.2.0",
     "@spectrum-css/icon": "^3.0.23",
     "@spectrum-css/radio": "^3.0.24",
+    "@spectrum-css/helptext": "^1.0.20",
```

Comment:
@pfulton we need to double-check this is the correct version after the beta-release